### ISSUE TITLE
Change the default of request.credentials

### ISSUE THREAD

**Issue by kevinsimper** [prisma-labs/graphql-playground#1192]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [ ] GraphQL Playground HTML
- [x] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

The default value of request.credentials is "omit". The differs from the default of Fetch changed so that it is `same-site`. 

https://github.com/whatwg/fetch/pull/585

It confuses when cookies is used for authorization.

Could it be changed to `same-site`?

**Comment by Emetrop** [comment]
Yes please, I just spent 2 hours with debugging code because of it. And it's happening with my every new project when I forget about it..

**Comment by kevinsimper** [comment]
Yeah I also spent a bit debugging 😅
